web/libs/editor/src/tags/custom/index.ts

```diff
@@ -11,7 +11,9 @@ Registry.addCustomTag("CustomInterface", {
   view: (CustomInterfaceComponent as unknown) as any,
   result: types.frozen(),
   resultName: "custominterface",
-  detector: (sn: any) => Boolean(sn?.value?.custominterface || sn?.custominterface),
+  // Do not detect regions by value for CustomInterface. Classification areas (global results)
+  // should deserialize as ClassificationArea, not CustomRegionModel, otherwise addResult is missing.
+  detector: () => false,
```

Comment:
if there are problems with false detection of classification we should fix the problem, not break the detection completely.